### PR TITLE
CCE / CIS numbers assigment to SLE 12/15 rules

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _ Assignment of CCE / CIS numbers or CIS numbers update covering SLE 12/15 rules which are part of the group CIS group 5.3 Configure PAM  _

#### Rationale:

- Some of the SLE 12/15 rules which belong to the CIS group 5.3 Configure PAM did not have CCE numbers assigned or correct CIS numebrs

#### Review Hints:

- _Review hints here. Replace this text. Don't use the italics format!_

- _Use this optional section to give any relevant information which could help the reviewer to more quickly and assertively understand and test the changes._

- _Good examples are useful commands, if it is better to review all commits together or in a suggested sequence, any relevant discussion in other PRs or issues, etc._

### CODE REVIEW

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

File: shared/references/cce-sle12-avail.txt [hunk 1]
```diff
@@ -1,35 +1,3 @@
...
-CCE-92276-5
-CCE-92277-3
-CCE-92278-1
-CCE-92279-9
-CCE-92280-7
-CCE-92281-5
-CCE-92282-3
-CCE-92283-1
-CCE-92284-9
-CCE-92285-6
```
Comment (marcusburghardt) [inline shared/references/cce-sle12-avail.txt:32]: You are probably removing more CCEs than necessary in this PR. Could you check, please?

File: shared/references/cce-sle15-avail.txt [hunk 2]
```diff
@@ -1,28 +1,3 @@
...
-CCE-91389-7
-CCE-91390-5
-CCE-91391-3
-CCE-91392-1
-CCE-91393-9
-CCE-91394-7
-CCE-91395-4
-CCE-91396-2
-CCE-91397-0
-CCE-91398-8
```
Comment (marcusburghardt) [inline shared/references/cce-sle15-avail.txt:25]: You are probably removing more CCEs than necessary in this PR. Could you check, please?

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

**Comment by rumch-se** [comment]
Hello @marcusburghardt 
I have done all necessary changes to this PR.
Have a nice day
Rumen

File: shared/references/cce-sle12-avail.txt [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+CCE-92276-5
+CCE-92277-3
+CCE-92278-1
+CCE-92279-9
+CCE-92280-7
+CCE-92281-5
```
Comment (marcusburghardt) [inline shared/references/cce-sle12-avail.txt:6]: Why are you including new CCEs in this list?

File: shared/references/cce-sle15-avail.txt [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
+CCE-91392-1
+CCE-91393-9
+CCE-91394-7
+CCE-91395-4
+CCE-91396-2
+CCE-91397-0
```
Comment (marcusburghardt) [inline shared/references/cce-sle15-avail.txt:6]: Why are you including new CCEs in this list?

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

File: shared/references/cce-sle12-avail.txt (hunk 3 above)
Comment (rumch-se) [reply shared/references/cce-sle12-avail.txt:6]: Hello @marcusburghardt 
I will check it out. 
Rumen

File: shared/references/cce-sle15-avail.txt (hunk 4 above)
Comment (rumch-se) [reply shared/references/cce-sle15-avail.txt:6]: Hello @marcusburghardt
I will check it out.
Rumen

**Comment by rumch-se** [comment]
Hello @marcusburghardt 
I corrected the issue with added CCE numebrs. I do believe that now everything will be OK.
Have a nice day
Rumen

**Review by marcusburghardt — APPROVED** [review]
@rumch-se , could you also check the #9888 and #9900, please?

**Pull request merged**